modules/katamari/src/main/ts/ephox/katamari/api/Regex.ts

```diff
@@ -0,0 +1 @@
+export const escapeRegExp = (text: string): string => text.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
```

Comment:
The `RegExp` part here feels unneeded. The module name is already called `Regex` so this could just be `escape` so that when it's used it's `Regex.escape`.

Reply:
Also could you please split this over 2 lines, as it increases readability. eg:
```suggestion
export const escapeRegExp = (text: string): string =>
  text.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
```

Reply:
It would probably also be a good idea to include a comment/reference where this came from:  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#escaping

Reply:
updated